addons/csi-vsphere/1-vsphere-csi-driver.yaml

```diff
@@ -1,11 +1,11 @@
-# Source: https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/v2.7.0/manifests/vanilla/namespace.yaml
+# Source: https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/v3.2.0/manifests/vanilla/namespace.yaml
```

Comment:
I don't feel too comfortable bumping the minor version at the moment, because we plan on cherry-pick this PR. Let's leave this for a follow up PR later on.

Reply:
You mean bumping the minor or major?

Reply:
You're right, major is what I meant.

Reply:
This was a misleading comment. The previous version was actually, going by the images.go, already on 3.1.2. If this was truly bumping the manifest from 2.7 to 3.x, much more would be changed. Ask me how I know.

Reply:
ACK, I left one comment related to the manifest.